README.md

```diff
@@ -1,4 +1,11 @@
-<h1 align="center"><a href="https://click-n-collect.shop"><img src="logo.svg" alt="Sylius Click 'N' Collect"></a></h1>
+<div style="display: flex; flex-direction:row; align-items: center;justify-content: space-around">
+    <a href="https://click-n-collect.shop" target="_blank">
+        <img src="docs/img/logo.svg" alt="Sylius Click 'N' Collect">
```

Comment:
`<img src="docs/img/logo.svg" alt="Click 'N' Collect">` & move logo in `docs/img/logo.svg`